core/src/test/java/org/apache/iceberg/catalog/CatalogTests.java

```diff
@@ -244,7 +244,7 @@ public void testLoadNamespaceMetadata() {
 
     assertThatThrownBy(() -> catalog.loadNamespaceMetadata(NS))
         .isInstanceOf(NoSuchNamespaceException.class)
-        .hasMessageStartingWith("Namespace does not exist: newdb");
+        .hasMessageStartingWith("Namespace does not exist: %s", NS);
```

Comment:
this is just a separate improvement to make sure that the namespace name isn't hardcoded in the expected error msg when setting `NS` to something else